### PR TITLE
Step-by-step process for logging on to rhino

### LINKED ISSUE #572: Step-by-step log on to rhino
**Proposed Domain**

Comp demo tutorial

**Content Summary**

For newbie users: simple run-down of how to log on to rhino following [these instructions](https://github.com/fredhutchio/scRNAseq/blob/master/remote_compute.md), with links to other demos that explain more complex versions

**Local Content Expert(s)**
Me, I wrote the content above and can convert.

### PR DESCRIPTION
**Pull request by k8hertweck**
Resolves #572

### CODE REVIEW

**Comment by k8hertweck** [comment]
@atombaby Thanks for this insight, it's very valid. I'll make some updates to make it clear `snail` isn't actually *required* but is the choice for this tutorial.

**Comment by k8hertweck** [comment]
Add additional links out to document more advanced options (like tmux and screen)

**Comment by k8hertweck** [comment]
Where's the additional info about the clusters going to live from now on?

**Comment by atombaby** [comment]
Most of that information I think is now in a few different places.  Those files are still extant, so if you'd like to merge this we can make sure to resolve links when we remove those pages.  Does that sound like a good approach?

**Comment by k8hertweck** [comment]
Finished updates as per comments from @atombaby .

Also ready for @smvelazquez to take a look!

File: _compdemos/first_rhino.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+    grabnode
+
+You'll now be asked a series of questions about what compute resources you would like to request.
+We'll describe requesting the smallest unit of resources here for demonstration purposes.
+When prompted:
+
+- type 1 for number of CPUs
+- hit `Enter` to select the default for memory
+- type 1 for number of days
+
```
Comment (smvelazquez) [inline _compdemos/first_rhino.md:125]: Insert an example of what the full command should look like

File: _compdemos/first_rhino.md [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+When requesting resources for your own work,
+choose appropriate selections to suit the needs of the task.
+
```
Comment (smvelazquez) [inline _compdemos/first_rhino.md:128]: Give an example of what other options users have, such as num of nodes, num of CPU's, time to run, etc.; follow with an example of the command on the command line

File: _compdemos/first_rhino.md (hunk 1 above)
Comment (vortexing) [reply _compdemos/first_rhino.md:125]: The full command actually IS `grabnode`.  It takes no arguments.

**Pull request merged**